cli/src/cmd/exec.rs

```diff
@@ -395,25 +425,22 @@ impl Cli {
 
                 Ok(ExitCode::SUCCESS)
             }
-            Subcommand::InstallPlugin(identifier, name, location, wasm) => {
-                // TODO: implement location handling
-                if let Some(wasm) = wasm {
-                    let wasm = tokio::fs::read(wasm).await?;
-                    let name = match name {
-                        Some(v) => v,
-                        None => "",
-                    };
-                    let location = match location {
-                        Some(v) => v,
-                        None => "",
-                    };
-                    let client = Client::new(self.host.as_str())?;
-                    let res = client
-                        .install_plugin(identifier, name.to_string(), location.to_string(), wasm)
-                        .await?;
-                    return Ok(ExitCode::SUCCESS);
-                }
-                Err(anyhow!("No wasm provided.  Note: the --location argument is currently not supported.  Please use --wasm instead for now."))
+            Subcommand::InstallPlugin(identifier, name, wasm) => {
+                let location = match &wasm {
+                    PathOrUrl::Path(v) => v.to_str().unwrap_or_else(|| ""),
+                    PathOrUrl::Url(v) => v.as_str(),
+                };
+                let name = match name {
+                    Some(v) => v,
+                    None => "",
+                };
```

Comment:
If there is no `name` here, do we assign a default in the DB or should there be some validation in the CLI? I haven't looked at the server, which may fail this command with some validation too...

Reply:
A plugin name is nullable in the DB.  This made me realize that it should be an optional field in the proto as well, so if it is not provided, it will be `None` in the application and `null` in the DB